cli-plugins/apps/how-to-guides.hbs.md

```diff
@@ -0,0 +1,611 @@
+# <a id='how-to-guides'>How-to-guides
+
+<!-- Intro to this point -->
+
+<!-- ## <a id='service-account-usage'> --service-account usage -->
+
+## <a id='custom-registry'> Custom registry credentials
+
+Apps CLI plug-in allows users to push images to their private registry by setting some flags when creating from local source code. 
```

Comment:
The `apps` CLI plug-in allows users to push images to their private registry by setting some flags when creating workloads from local source code.